frontend/packages/data-portal/app/graphql/getBrowseDepositions.server.ts

```diff
@@ -34,44 +37,36 @@ const GET_DEPOSITIONS_DATA_QUERY = gql(`
         corresponding_author_status
       }
 
-      annotations_count: runs {
-        tomogram_voxel_spacings {
-          annotations_aggregate {
-            aggregate {
-              count
-            }
-          }
+      annotations_aggregate {
+        aggregate {
+          count
         }
       }
 
-      datasets_count: runs_aggregate {
-        aggregate {
-          count
+      annotations(distinct_on: object_name) {
+        object_name
+      }
+
+      shape_types: annotations {
+        files(distinct_on: shape_type) {
```

Comment:
you should be able to avoid having to do the `flatMap()`s if you query `annotation_files` at the root

Reply:
this doesnt work since i need shape types for each deposition entry :/